README.md

```diff
@@ -2,71 +2,97 @@
 
 This repository contains all the necessary files to recreate an experiment that compares our implementation of Code Climate's quality model against the actual Code Climate tool.
 Additionally, it contains the actual data that we have used in our comparison of the two quality models.
-It consists of three parts:
+It consists of four components:
 
-1. A bash script to run our tool on all the tags of a given git repository
-2. A python program to run code climate on all the tags of a given git repository, outputting in a format compatible with our tool (this can also be run through a bash script)
-3. A python program that can visualize the outputs of the two aforementioned parts using bokeh
+1. A bash script that transforms a local git repository into a folder per tag of that git repository
+2. A python program that will run [modu](https://github.com/parameterIT/tool/) on each sub-directory of a given directory (this will be paired with script (1))
+3. A python program that runs code climate on each sub-directory of a given directory (this will be paired with script (1))
+4. A python program that compares the results of (2) and (3) visually on line graphs
 
-## Set-up 
+## Requirements
 
-The tools use your local installation of git, so make sure that it is set-up available in your PATH.
-Second, tool (1) requires that the git repository you wish to execute `byoqm` on is a sibling folder to this folder.
-Make sure that you have the following folder structure:
+- Many of the provided scripts and programs make use of your local installation of _git_, so make that it is set-up and available in your PATH.
+- Component (2) requires a local installation of _modu_. The project's [README](https://github.com/parameterIT/tool/blob/main/README.md) describes how to set it up
```

Comment:
What is Component (2)? Is it a reference to point 2 in the above section?

Reply:
Yes, does it need to be made more clear?